examples/show_hids.py

```diff
@@ -3,8 +3,27 @@
 
 #
 """
-Show all HID devices information
+Show all HID devices' information
+
+Written By Rene Aguirre (https://github.com/rene-aguirre) ~2016
```

Comment:
This is basically a changelog, I'd prefer to move these lines to a changelog comment section.